test/controller.e2e-spec.ts

```diff
@@ -118,3 +119,37 @@ describe.each`
     });
   });
 });
+describe('SkipIf suite', () => {
+  it('should skip throttling if skipIf returns true', async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [ControllerModule],
+      providers: [
+        {
+          provide: APP_GUARD,
+          useClass: ThrottlerGuard,
+        },
+      ],
+    })
+      .overrideProvider(THROTTLER_OPTIONS)
+      .useValue({
+        skipIf: () => true,
+        limit: 5,
+      })
+      .compile();
```

Comment:
```suggestion
    .overrideProvider(THROTTLER_OPTIONS)
    .useValue({
      skipIf: () => true,
      limit: 5,
    })
    .compile();
```
```suggestion
      .overrideProvider(THROTTLER_OPTIONS)
      .useValue({
        skipIf: () => true,
        limit: 5,
      })
      .compile();
```

Reply:
I think there is some indentation to fix here, but nothing problematic 😄

Reply:
`yarn format` shows no changes need to be made

Reply:
It will be github that has different indentation, however no problem 👍